modules/mysql/variables.tf

```diff
@@ -500,3 +507,13 @@ variable "database_integration_roles" {
   default     = []
 }
 
+variable "connection_pool_config" {
```

Comment:
@ichbinfrog is it a required variable? It is not required in postgres module.

Reply:
hmm, nope, I've updated it for the mysql module as well